appcues/src/main/java/com/appcues/trait/appcues/CarouselTrait.kt

```diff
@@ -31,7 +30,6 @@ internal class CarouselTrait(
         const val TYPE = "@appcues/carousel"
     }
 
-    @OptIn(ExperimentalFoundationApi::class)
```

Comment:
as noted in PR description - HorizontalPager stuff is no longer marked experimental, so this is removed